technical guides/Walls.md

```diff
@@ -0,0 +1,258 @@
+# Breaking Down Walls - Teeth and Fortifications in Beyond All Reason
+<sub></sup>(Author: TitanShadow12)</sup></sub>
+
+Walls in Beyond All Reason are a cheap and simple way to shore up gaps in defenses or mess with enemy pathfinding. The width of most maps in BAR along with the slowness and risk of using constructors to build walls make these structures seem like a very niche form of passive defense, but with some cleverness and proper placement, even the tiny Dragon's Teeth can make an impact on your frontline.
+
+## Wall Stats
+The following 3 walls will be covered in this guide.
+
+
+| Name | Unitdefname | HP | Regen | Metal Cost | Energy Cost | Buildtime | Constructed By |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| **Dragon's Teeth (T1 Walls)** | armdrag / cordrag | 2800 | 4/s | 8 | 0 | 255 | All T1 cons, minelayers, commanders |
+| **Shark's Teeth (T1 Naval Walls)** | armfdrag / corfdrag | 4450 | 4/s | 15 | 0 | 400 | T1 vehicle amphib, hover, seaplane, and ship cons; commanders |
+| **Fortification Walls (T2 Walls)** | armfort / corfort | 8900 | 12/s | 40 | 500 | 900 | All T2 constructors (except ship), Twitcher, Consul |
+
+## Wall Basics
+Let's set some ground rules you'll find common among walls in Beyond All Reason before we get into strategies and technical bits.
+- Walls obstruct pathfinding and unit movement
+- Some units can crush walls in their path and do not have their pathfinding interrupted
+- Walls cost 0 energy and a small amount of metal
+- Walls have extremely high HP for their cost
+- Walls are built one unit at a time, and all walls have the same footprint size (verify for naval walls). Building walls by holding shift to place lines is much more effective than laying them down one at a time

```

Comment:
Make sure to verify that part for naval walls 😁

Reply:
Confirmed the following matches between armdrag and armfdrag:
		footprintx = 2,
		footprintz = 2,

Reply:
```suggestion
- Walls are built one unit at a time, and all walls have the same footprint size. Building walls by holding shift to place lines is much more effective than laying them down one at a time

```